app/src/main/java/com/example/demo/MainActivity.kt

```diff
@@ -15,6 +15,13 @@ import com.example.demo.ui.theme.DemoTheme
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty
<!-- __reviewdog__:ChAzZDZmOTdhMzljYTUxODNiEgVzYXJpZg== -->